iso690-numeric-brackets-cs.csl

```diff
@@ -420,8 +420,15 @@
       <key variable="citation-number"/>
     </sort>
     <layout prefix="[" suffix="]" delimiter=", ">
-      <text variable="citation-number"/>
-      <text variable="locator" prefix=" s. "/>
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text variable="citation-number"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </group>
     </layout>
```

Comment:
Okay, I see what you are doing, but this can be simplified. Try this (you may want to indent it correctly):
```
     <layout prefix="[" suffix="]" delimiter=", ">


          <text variable="citation-number"/>

        <group delimiter=" ">
          <label variable="locator" form="short"/>
          <text variable="locator"/>
        </group>

     </layout>
```
(A group with only one element is not useful and the group inside the layout is not necessary.)